backend/infrahub/events/proposed_change_action.py

```diff
@@ -150,6 +150,33 @@ class ProposedChangeRejectionRevokedEvent(ProposedChangeReviewRevokedEvent):
     event_name: ClassVar[str] = f"{EVENT_NAMESPACE}.proposed_change.rejection_revoked"
 
 
+class ProposedChangeApprovalsRevokedEvent(ProposedChangeEvent):
+    reviewer_account_ids: list[str] = Field(..., description="IDs of accounts whose approval was revoked")
+    reviewer_account_names: list[str] = Field(..., description="Names of accounts whose approval was revoked")
```

Comment:
I wonder if a `dict[str,str]` would be better here. It doesn't really matter from a Prefect point of view as we're iterating over each list to match them. But these fields would also be included in the payload of the event, as such a receiver would have to combine them to match the ids to the names. What do you think?